### PR TITLE
Updated PR Template

### LINKED ISSUE #698: Update the PR template
**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by sachin235**
Fixes #698

Changes: 
Rectified grammatical issues in the PR template and added details regarding screenshot.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this issue and if possible provide links referring the changes.]
```
Comment (neeraj3029) [inline .github/PULL_REQUEST_TEMPLATE]: `made in this Pull Request` would be more precise

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 
 Surge Deployment Link: https://pr-${TRAVIS_PULL_REQUEST}-fossasia-susi-accounts.surge.sh
 
-Screenshots for the change:
+Screenshots for the change: [Add here screenshots depicting the changes you made in order to resolve this issue.]
```
Comment (neeraj3029) [inline .github/PULL_REQUEST_TEMPLATE]: I think the `Screenshots for the change` explains itself, no need to explain it more.

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (sachin235) [reply .github/PULL_REQUEST_TEMPLATE]: Yes done.

File: .github/PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (sachin235) [reply .github/PULL_REQUEST_TEMPLATE]: @neeraj3029 Thanks for review
I've made the changes as suugested.

**Comment by sachin235** [comment]
@neeraj3029 
Please review this after the changes as suggested by you, and let me know if anything else needs to be modified.

**Pull request merged**